docs/format/tile.rst

```diff
@@ -16,6 +16,15 @@ General notes:
 tilegrid.json
 -------------
 
+The file `tilegrid.json` contains lists of all tiles in the ROI and the configuration segments formed by those tiles. It also documents the membership relationship of tiles and segments.
+
+For each segment this contains the configuration frame base address, and the word offset within the frames, as well as the number of frames for the segment and number of occupied words in each frame.
+
+For each tile this file contains the tile type, grid X/Y coordinates for the tile, and sites (slices) within the tile.
+
+Note that pairs of INT-tiles and non-INT-tiles form a segment. The same type-name (e.g. `CLBLL_R` is used for the non-INT tile within the segment, and the segment itself). So for example, a `CLBLL_R` segment contains a `CLBLL_R` tile and an `INT_R` tile.
```

Comment:
Delete this sentence, segments are no longer used.

Reply:
Removed.